setup.py

```diff
@@ -8,6 +8,10 @@
 with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+# Get requirements from requirements.txt file
+reqs = [line.strip() for line in open('requirements.txt').readlines()]
+requirements = list(filter(None, reqs))
+
 setup(
     # Application name:
     name="nidmviewer",
```

Comment:
I don't know if that will fail. Only think I can think of is different syntax between setup.py and requirements.txt when it comes to specifying versions.